### PR TITLE
Add scheduled CLU task creation to Bitbucket provider startTesting task

### PR DESCRIPTION
**Pull request by aaronbauman**
Depends on the `clu` task getting defined in bitbucket-pipelines.yml 
https://github.com/pantheon-systems/example-drops-8-composer/pull/289

### CODE REVIEW

File: src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php [hunk 1]
```diff
@@ -109,10 +109,32 @@ public function startTesting(CIState $ci_env)
...
+            // Enable Pipelines for the project.
             $this->api()->request("$repoApiUrl/pipelines_config", $data, 'PUT');
         } catch (\Exception $e) {
             $this->logger->error($e->getMessage());
         }
+
+        // @TODO: verify that the custom "clu" task exists in yml first?
+        $data = [
+          'enabled' => true,
+          'cron_pattern' => '0 0 4 * * ? *',
```
Comment (ataylorme) [inline src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php]: This cron pattern seems odd. Is this what worked for you?

Comment (ataylorme) [reply src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php]: I was expecting a more regular pattern like `0 0 * * *` for "every day at midnight"

Comment (aaronbauman) [reply src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php]: Yeah, it's crazy. I reverse engineered this statement by calling the GET endpoint to see what a manually created job looks like.

Based on that, I think maybe it follows Jira's synatx: https://confluence.atlassian.com/jiracorecloud/constructing-cron-expressions-for-a-filter-subscription-765593786.html

Comment (ataylorme) [reply src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php]: Okay, so following that example `0 0 4 * * ? *` is every day at 4 am. The `?` is needed for _day of the week_ to "Specify no particular value."

Comment (ataylorme) [reply src/ServiceProviders/CIProviders/BitbucketPipelines/BitbucketPipelinesProvider.php]: Maybe this is a good opportunity to open a PR to the BitBucket Pipelines API docs for Hacktoberfest

**Comment by ataylorme** [comment]
Tests running in #304

**Pull request closed without merging**